nais/dev-fss.yml

```diff
@@ -37,9 +37,9 @@ spec:
   vault:
     enabled: false
   env:
-    - name: HRP_k9_selvbetjening_oppslag
-      value: http://k9-selvbetjening-oppslag
     - name: HRP_frisinn_mottak
       value: http://frisinn-mottak
     - name: HRP_k9_dokument
       value: http://k9-dokument.dusseldorf
+    - name: HRP_k9_selvbetjening_oppslag
```

Comment:
Hvordan blir det for tjenester som kaller dette endepunktet da?
De har jo ikke spesifisert dusseldorf i sin url når de kaller den og vil feile når du har fjernet den med default namespace. 

Du må ha støtte for begge i overganger. Slik at de tjenestene som kaller denne kan migrere til den nye adressen.

Reply:
Åja hm jeg trodde det var "name" som blir brukt? Feks spør frisinn mot "https://api-gw-q1.oera.no/helse-reverse-proxy/k9-selvbetjening-oppslag".
Blir det ikke håndtert slik her? https://github.com/navikt/helse-reverse-proxy/blob/master/src/main/kotlin/no/nav/helse/Environment.kt#L21

Name blir mappet til url(value)

Reply:
https://logs.adeo.no/app/discover#/doc/96e648c0-980a-11e9-830a-e17bbd64b4db/logstash-apps-test-001536?id=Km4LV3oBMwa38frMqdYN

Reply:
Åja, trodde det var value som ble brukt. Da burde det holde :D

Reply:
Har også teste i dev med frisinn som bruker den url, det fungerte

Reply:
Nice nice. Antok at det var value som ble brukt basert på namingen :p

Reply:
Ja var ikke helt intuitivt kanskje :P